Changelog.md

```diff
@@ -1,6 +1,14 @@
+## Version 1.4.0
+## New feature
+* Added support for nRF52820
+* Added support for modem UART DFU
+
+## Bugfixes
+* Fixed end address for regions displayed
+
 ## Version 1.3.1
 ## Bugfixes
-- Fix programming nRF51 devices with UICR #156
+* Fixed programming nRF51 devices with UICR #156
 
 ## Version 1.3.0
 ### New features
```

Comment:
To continue unifying phrases: s/Supported/Added support for/

Reply:
Good point!